example/docker-compose.yml

```diff
@@ -25,12 +25,9 @@ services:
     ports:
       - "${WIKIBASE_PORT}:80"
     volumes:
-      ## This shares the configuration with jobrunner
-      - shared:/var/www/html/:rw
-
-      ## Uncomment this next line to mount your own LocalSettings.php file
+      - shared-localsettings:/var/shared-localsettings
+      ## Uncomment this next line to mount your own LocalSettings.php file overrides
       #- ./LocalSettings.php:/var/www/html/LocalSettings.d/LocalSettings.override.php
```

Comment:
is this legit advice? Won't this cause a deviation between the settings on the job runner and the settings on the web server?

Reply:
This comment exists for both the wikibase and job runner containers.
So the expectation would be a user would mount any additional file into both containers

Reply:
Altered in https://github.com/wmde/wikibase-release-pipeline/pull/359/commits/75631dd9db9e8782979352d3340af0742fdb1b10 to only appear in 1 place